library/public/camera.h

```diff
@@ -44,6 +44,7 @@ class F3D_EXPORT camera
   virtual vector3_t getViewUp() = 0;
   virtual void getViewUp(vector3_t& up) = 0;
   virtual camera& setViewAngle(const angle_deg_t& angle) = 0;
+  virtual camera& rotateCamera(const angle_deg_t& angle) = 0;
```

Comment:
I think you can use `roll` instead of adding a method

Reply:
yeah much better